spec.md

```diff
@@ -270,7 +270,19 @@ An asynchronous response triggers the platform marketplace to poll the endpoint
 
 #### Blocking Operations
 
-The marketplace MUST ensure that service brokers do not receive requests for an instance while an asynchronous operation is in progress. For example, if a broker is in the process of provisioning an instance asynchronously, the marketplace can not allow any update, bind, unbind, or deprovision requests to be made through the platform. A user who attempts to perform one of these actions while an operation is already in progress MUST receive a `400 Bad Request` response with the error message: `Another operation for this service instance is in progress`.
+This specification places no requirement on brokers with respect to their
```

Comment:
I don't think the first sentence of this paragraph agrees with the second sentence, but I get the idea of the first.  Is the first sentence better put as: "brokers are not required to support parallel requests on the same set of resources".  The requirement is actually the opposite of supporting parallel requests - broker's are required to reject parallel requests.